lib/handshake_server13.ml

```diff
@@ -342,7 +342,8 @@ let answer_client_hello ~hrr state ch raw =
 
       let session =
         let common_session_data13 = { session'.common_session_data13 with master_secret = master_secret.secret } in
-        { session' with common_session_data13 ; master_secret (* TODO ; exporter_secret *) }
+        let exporter_master_secret = Handshake_crypto13.exporter session.master_secret log in
```

Comment:
as mentioned above, the exporter secret only goes until the server finished, this is why we can nicely compute it here (and the resumption secret is computed at another place to include the client finished).